ingest/processing.py

```diff
@@ -135,7 +135,6 @@ def dataset2fgb(fgb_dir: str = None,
                 '-preserve_fid',
                 # '-skipfailures',
```

Comment:
@iferencik I think you have deleted wrong line if this PR title is to use skipfailures option